go/arrow/internal/arrjson/arrjson_test.go

```diff
@@ -36,12 +42,11 @@ func TestReadWrite(t *testing.T) {
 			mem := memory.NewCheckedAllocator(memory.NewGoAllocator())
 			defer mem.AssertSize(t, 0)
 
-			f, err := ioutil.TempFile("", "arrjson-")
+			f, err := ioutil.TempFile(tempDir, "arrjson-")
```

Comment:
or use the test name to derive a file name?

Reply:
@sbinet Well, I prefer the test function name. And maybe add "go-arrow-" prefix will be better.

Reply:
fair enough.